pkg/crane/append.go

```diff
@@ -30,17 +30,23 @@ import (
 func init() { Root.AddCommand(NewCmdAppend()) }
 
 func NewCmdAppend() *cobra.Command {
-	var output string
+	var baseRef, newTag, newLayer, outFile string
 	appendCmd := &cobra.Command{
 		Use:   "append",
 		Short: "Append contents of a tarball to a remote image",
-		Args:  cobra.ExactArgs(3),
+		Args:  cobra.NoArgs,
 		Run: func(_ *cobra.Command, args []string) {
-			src, dst, tar := args[0], args[1], args[2]
-			doAppend(src, dst, tar, output)
+			doAppend(baseRef, newTag, newLayer, outFile)
 		},
 	}
-	appendCmd.Flags().StringVarP(&output, "output", "o", "", "Path to new tarball of resulting image")
+	appendCmd.Flags().StringVarP(&baseRef, "base", "b", "", "Name of base image to append to")
+	appendCmd.Flags().StringVarP(&newTag, "new_tag", "t", "", "Tag to apply to resulting image")
+	appendCmd.Flags().StringVarP(&newLayer, "new_layer", "f", "", "Path to tarball to append to image")
```

Comment:
Why `-f`? As in file?

Reply:
Yeah. Open to suggestions if you're opposed.